python/py-CacheControl/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-CacheControl
+version             0.12.6
+revision            0
+categories-append   devel
+platforms           darwin
+license             Apache-2
+supported_archs     noarch
+
+python.versions     27 34 35 36 37 38
```

Comment:
Should we have _new_ ports depending on Python 2.7?

Reply:
I don't think we should. Only 37 and 38 should be here.